### PR TITLE
[SWOC'24] Factorial Calculator Project Ready

### LINKED ISSUE #77: [SWOC'24] Factorial Calculator
### Prerequisites

- [X] I am an Open Source Contributor
- [X] I checked to make sure that this issue has not already been filed

### Description

- Enter the number, project will return the factorial of that entered number instantly.
- Card and colouring effect to attract users.
- Animated effects.

### Screenshots

This calculator will help user to calculate large number factorials in a very little period of time.

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/CalcDiverse/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.

### PR DESCRIPTION
**Pull request by Avdhesh-Varshney**
# Fixes Issue🛠️

Closes #77 

# Description👨‍💻 

- Factorial calculator is ready.
- Responsive for small devices too.
- Glassmorphism effect of the card.
- Best designing of the background.

# Type of change📄

- [x] New feature (non-breaking change which adds functionality)

# How this has been tested✅

# Checklist✅ 

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added demonstration in the form of GIF/video file
- [x] I am an Open Source Contributor

# Screenshots/GIF📷

image

### CODE REVIEW

**Comment by Avdhesh-Varshney** [comment]
@Rakesh9100 please review this pr and requested to merge this pr under SWOC'24

**Comment by Rakesh9100** [comment]
Pls resolve the conflicts @Avdhesh-Varshney

File: Calculators/Factorial-Calculator/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## **ScreenShot 📸**
+
+![image](https://github.com/Rakesh9100/CalcDiverse/assets/114330097/1b5826af-c3ce-4c76-bde4-f090fbb2e4c7)
+
+<br>
+
+## **Developed By 👦**
+
+[Avdhesh Varshney](https://github.com/Avdhesh-Varshney)
+
```
Comment (Rakesh9100) [inline Calculators/Factorial-Calculator/README.md:30]: Pls remove this `Developed By` section from Readme @Avdhesh-Varshney

File: Calculators/Factorial-Calculator/index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+<body>
+    <div class="box1"></div>
+    <div class="box2"></div>
+    <!-- Main container of the application -->
+    <div class="container">
+        <!-- Main heading of the application -->
+        <h1>Factorial Calculator</h1>
+
+        <!-- Creator of this application -->
+        <p>Created By <a href="https://github.com/Avdhesh-Varshney">Avdhesh Varshney</a></p>
```
Comment (Rakesh9100) [inline Calculators/Factorial-Calculator/index.html:29]: Pls remove `Created By` from the calculator @Avdhesh-Varshney

**Comment by Rakesh9100** [comment]
Resolve the conflicts pls @Avdhesh-Varshney

**Comment by Avdhesh-Varshney** [comment]
@Rakesh9100 please again review this pr and conflicts are resolved!

**Comment by Rakesh9100** [comment]
Good job, Merged!! 🥳 @Avdhesh-Varshney

**Pull request merged**